README.md

```diff
@@ -35,15 +35,24 @@ If you can't use GitHub, you can use other providers:
    
         npm install
 
-4. Generate a GitHub access token by going to <https://github.com/settings/tokens/new>.  The access token should have the `repo` scope/permission.  Once you have the token, assign it to an environment variable (on macOS/linux):
+4. Generate a GitHub access token by going to <https://github.com/settings/tokens/new>.  The access token should have the `repo`            scope/permission.  Once you have the token, assign it to an environment variable 
 
-        export GH_TOKEN="<YOUR_TOKEN_HERE>"
+    (on macOS/linux):
+
+            export GH_TOKEN="<YOUR_TOKEN_HERE>"
+
+    (on Windows, run in powershell):
+
+            [Environment]::SetEnvironmentVariable("GH_TOKEN","<YOUR_TOKEN_HERE>","User")
+
+    make sure to restart IDE/Terminal to inherit latest env variable.
 
 5. Publish with the `publish.sh` script:
 
         ./publish.sh
-
-6. Release the release on GitHub by going to <https://github.com/iffy/electron-updater-example/releases>, editing the release and clicking "Publish release."
+    In case of Windows, directly run the command: ```node_modules/.bin/build --win -p always```
```

Comment:
Can you think of a way to merge this `node_modules/.bin/build --win -p always` command with `./publish.sh` so that step 5 could say:

5. Publish with `npm run publish`.

Reply:
@develar would it work to just run `node_modules/.bin/build -p always` without specifying a platform?

Reply:
sorry for late reply.. timezone difference. I see you've figured it out. it would work.